.github/workflows/playwright.yml

```diff
@@ -67,4 +67,4 @@ jobs:
           path: |
             examples/playwright/test-results/
             examples/playwright/playwright-report/
-          retention-days: 2
+          retention-days: 7
```

Comment:
why are we increasing this by so much?

Reply:
Sometimes (sickness/holiday) it takes some time to go back to the failed PR and check the results. In my case 5 days would be probably enough thought.